daos/mysql/Items.php

```diff
@@ -296,12 +299,32 @@ public function get($options = array()) {
         $this->hasMore = count($result);
 
         // get items from database
-        return \F3::get('db')->exec('SELECT
-                    items.id, datetime, items.title AS title, content, unread, starred, source, thumbnail, icon, uid, link, updatetime, author, sources.title as sourcetitle, sources.tags as tags
-                   FROM '.\F3::get('db_prefix').'items AS items, '.\F3::get('db_prefix').'sources AS sources
-                   WHERE items.source=sources.id AND '.$where_sql.'
-                   ORDER BY items.datetime '.$order.'
-                   LIMIT ' . $options['items'] . ' OFFSET '. $options['offset'], $params);
+        $select = 'SELECT
+            items.id, datetime, items.title AS title, content, unread, starred, source, thumbnail, icon, uid, link, updatetime, author, sources.title as sourcetitle, sources.tags as tags
+            FROM '.\F3::get('db_prefix').'items AS items, '.\F3::get('db_prefix').'sources AS sources
+            WHERE items.source=sources.id AND';
+        $order = 'ORDER BY items.datetime,items.id '.$order;
```

Comment:
This should probably be `'ORDER BY items.datetime ' . $order . ', items.id ' . $order`